CHANGELOGS.md

```diff
@@ -0,0 +1,54 @@
+# 2022-05-04
```

Comment:
Should we remove the date since we merged into one file now?

Reply:
yeah sorry forget to update that